test/infrastructure/docker/SECURITY_CONTACTS

```diff
@@ -0,0 +1,16 @@
+# Defined below are the security contacts for this repo.
```

Comment:
We can likely remove this in favor of the SECURITY_CONTACTS at the root of the repo.

Reply:
ack